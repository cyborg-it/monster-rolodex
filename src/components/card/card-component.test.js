import renderer from 'react-test-renderer';
import { Card } from './card.component';

const demoCardData = {
    id: 1,
    name: "Leanne Graham",
    username: "Bret",
    email: "Sincere@april.biz"
}

describe('snapshot test suites for Card component', () => {
    it('render Card component', () => {
        const tree = renderer.create(<Card id={demoCardData.id} monster={demoCardData} />).toJSON();
        expect(tree).toMatchSnapshot()
    });
})