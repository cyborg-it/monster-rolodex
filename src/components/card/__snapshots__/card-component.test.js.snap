// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test suites for Card component render Card component 1`] = `
<div
  className="card-container"
>
  <img
    alt=""
    src="https://robohash.org/1?set=set2&size=180x180"
  />
  <h2>
    Leanne Graham
  </h2>
  <p>
     
    Sincere@april.biz
     
  </p>
</div>
`;
